store/driver/txn/unionstore_driver.go

```diff
@@ -146,6 +146,14 @@ func getTiDBKeyFlags(flag tikvstore.KeyFlags) kv.KeyFlags {
 	if flag.HasNeedLocked() {
 		v = kv.ApplyFlagsOps(v, kv.SetNeedLocked)
 	}
+
+	if flag.HasAssertExist() {
+		v = kv.ApplyFlagsOps(v, kv.SetAssertExist)
+	} else if flag.HasAssertNotExist() {
+		v = kv.ApplyFlagsOps(v, kv.SetAssertNotExist)
+	} else if flag.HasAssertion() {
```

Comment:
Is this `HasAssertUnkonw` ?

Reply:
HasAssertion = HasAssertExist | HasAssertNotExist | HasAssertUnknown. So using HasAssertion here is equivalent to HasAssertUnknown. But maybe the latter choice looks more comprehensible. I'll update it later.